src/commands/lambda/flare.ts

```diff
@@ -771,6 +784,75 @@ export const sleep = async (ms: number) => {
   await new Promise((resolve) => setTimeout(resolve, ms))
 }
 
+/**
+ * Get the framework used based on the files in the directory
+ * @returns the framework used or undefined if no framework is found
+ */
+export const getFramework = () => {
+  const files = new Set(fs.readdirSync(process.cwd()))
+
+  // Check for serverless framework
+  const serverlessFiles = ['serverless.yml', 'serverless.yaml']
```

Comment:
nit: repeating constants, we need to find a way to get a subset of these from the `PROJECT_FILES`

Reply:
Fixed, accomplished using the array spread operator:
```
export const FRAMEWORK_MAPPINGS = new Map([
  ['serverless.yaml', Frameworks.ServerlessFramework],
  ['serverless.yml', Frameworks.ServerlessFramework],
  ...
])

// Project files to search for in Flare
export const PROJECT_FILES = [
  ...FRAMEWORK_MAPPINGS.keys(),
  // Datadog CloudFormation Template
  'datadog-cloudfomation-macro.yaml',
  ...
]
```

Reply:
Left a comment